### PR TITLE
Created feedbacks view

### PR DESCRIPTION
**Pull request by dracdev**
- Implemented front-end view for Observing Feedbacks
- Implemented repository IncludeFields method

### CODE REVIEW

**Comment by dracdev** [comment]
[[E3.3] As a system administrator I want to observe feedbacks so that I could better manage the userbase. [2]](https://trello.com/c/0oQjBsrq/19-e33-as-a-system-administrator-i-want-to-observe-feedbacks-so-that-i-could-better-manage-the-userbase-2)

**Review by FmasterofU — DISMISSED** [review]

File: HospitalFrontend/src/views/ObserveFeedback.vue [hunk 1]
```diff
@@ -1,10 +1,45 @@
...
+  components:
+  {
+     FeedbackItem
+  },
+  methods: 
+  {
+
+    getAllFeedbacks:function()
+    {
+      axios.get("http://localhost:5290/feedback").then((response) =>
```
Comment (BoKna031) [inline HospitalFrontend/src/views/ObserveFeedback.vue]: Defining a direct path can cause an error to other developers @dracdev

**Review by BoKna031 — CHANGES_REQUESTED** [review]
Need to fix port issue

File: HospitalFrontend/src/views/ObserveFeedback.vue (hunk 1 above)
Comment (dracdev) [reply HospitalFrontend/src/views/ObserveFeedback.vue]: Fixed in collaboration with other developers

**Review by BoKna031 — APPROVED** [review]
In agreement with the web team the port problem has been fixed temporarily.

**Review by FmasterofU — APPROVED** [review]

**Comment by FmasterofU** [comment]
[PR Created feedbacks view](https://trello.com/c/DpJpp81x/53-pr-created-feedbacks-view)

**Pull request merged**